llvm/lib/Target/Mips/AsmParser/MipsAsmParser.cpp

```diff
@@ -2307,7 +2319,42 @@ bool MipsAsmParser::processInstruction(MCInst &Inst, SMLoc IDLoc,
 
   bool FillDelaySlot =
       MCID.hasDelaySlot() && AssemblerOptions.back()->isReorder();
-  if (FillDelaySlot)
+
+  // Get previous instruction`s forbidden slot attribute and
+  // whether set reorder.
+  // This 'CurForbiddenSlotAttr' is a global variable.
```

Comment:
It is not `global` variable, instead it is a class level variable.

Reply:
OK, I would modify these comments.